HighwayPatrol.cpp

```diff
@@ -25,10 +28,17 @@ void HighwayPatrol::pullOver( Vehicle* v, bool willArrest, Highway* h )
 {
     std::cout << "\n\n";
     std::cout << name << ": vehicle is traveling " << v->speed - h->speedLimit << " miles per hour over the speed limit" << std::endl;
+
+
     if( willArrest )
     {
-        //print the vehicle type in this std::cout between "THE [" and "] PULL". 
-        std::cout << name << ": YOU IN THE [ " << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
+        std::string type = "wrong vehicle type";
+
+        if (dynamic_cast<Car*>(v)) type = "car";
+        else if (dynamic_cast<Motorcycle*>(v)) type = "motorcycle";
+        else if (dynamic_cast<SemiTruck*>(v)) type = "semitruck";
```

Comment:
You should prefer to put `if/else` bodies on their own lines, even if they're one line.  
Or, if they are one line, align them all vertically and use `{ }`

- [ ] formatting